src/Codeception/Lib/InnerBrowser.php

```diff
@@ -105,7 +105,18 @@ public function _findElements($locator)
     public function _request($method, $uri, array $parameters = [],  array $files = [], array $server = [], $content = null)
     {
         $this->clientRequest($method, $uri, $parameters, $files, $server, $content, false);
-        return $this->getRunningClient()->getInternalResponse()->getContent();
+        return $this->_getResponseContent();
+    }
+
+    /**
+     * Returns content of the last response
+     * 
```

Comment:
It would be good to have a usage example here:

basic one:

```php
$content = $this->getModule('{{MODULE_NAME}}')->_getResponseContent();
```

or something like real example:

```php
function seeMobileReady() {
    $content = $this->getModule('{{MODULE_NAME}}')->_getResponseContent();
    $this->assertRegex('~<meta (/*?)name="viewport"~', $content);
}
```